tests/test_build.py

```
@@ -91,13 +93,18 @@ def test_build_works_WHEN_supporting_pep517(self) -> None:
 		build_arguments = [
 			f"{str(sys.executable)} -m coverage run", "-p", "-m", "build", "--sdist", "--wheel",
 		]
+		# Temporarily relax the default umask (to allow creation of venv files)
+		original_umask = os.umask(0o027)  # Temporarily set the umask
 		# Build the source distribution
-		theBuildtxt = context.checkPythonCommand(build_arguments, stderr=subprocess.STDOUT)
+		try:
+			theBuildtxt = context.checkPythonCommand(build_arguments, stderr=subprocess.STDOUT)
+		finally:
+			os.umask(original_umask)  # Restore the original umask
+			self.assertIsNotNone(theBuildtxt, f"Failed with {build_arguments} in relaxed state")
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertIsNotNone`

Replace `assertIsNotNone(...)` with `assert ...`

(PT009)